src/modules/Crowbar.rb

```diff
@@ -118,6 +93,13 @@ def json2hash(file_name)
       ret
     end
 
+    # read given yaml file and return the content as a map
+    def yaml2hash(file_name)
+      ret = YAML.load(File.read(file_name))
```

Comment:
Hm, this fills nils for url's. Wondering if
1. we should have empty string values in /opt/dell/crowbar_framework/config/repos-cloud.yml
2. we should convert nils to empty strings in yaml2hash
3. we should treat nils in the UI the same way we treat empty string (this might be easiest, but I'm not sure if correct)